frontend/src/components/SearchBar/FilterBar.scss

```diff
@@ -9,4 +8,11 @@
       margin: 0;
     }
   }
+  .vl {
```

Comment:
What is `vl`?

Reply:
vertical line to separate the buttons of different functionality! based off of the UI